fs/vfs/fs_uio.c

```diff
@@ -39,15 +39,15 @@
  ****************************************************************************/
 
 /****************************************************************************
- * Name: uio_total_len
+ * Name: uio_resid
  *
  * Description:
- *   Return the total length of data in bytes.
+ *   Return the remaining length of data in bytes.
  *   Or -EOVERFLOW.
  *
  ****************************************************************************/
 
-ssize_t uio_total_len(FAR const struct uio *uio)
+ssize_t uio_resid(FAR const struct uio *uio)
```

Comment:
uio_remain?

Reply:
"resid" is the term used by BSDs.

Reply:
ok. let me think i bit.

Reply:
i added uio_resid and (commented out) uio_offset